templates/sfn/ingest_sfn_definition.json.tpl

```diff
@@ -440,22 +445,29 @@
       ],
       "Default": "Wait 5 minutes before getting status"
     },
-    "Job Failed": {
-      "Type": "Fail",
-      "Cause": "AWS Batch Job Failed",
-      "Error": "'Check workflow status' task returned Failed"
-    },
     "Map over each assetId and reconcile": {
       "Type": "Map",
-      "ItemsPath": "$.contentAssets",
+      "Label": "MapOverEachAssetIdAndReconcile",
+      "ItemReader": {
+        "Resource": "arn:aws:states:::s3:getObject",
+        "ReaderConfig": {
+          "InputType": "JSON"
+        },
+        "Parameters": {
+          "Bucket": "$.assets.bucket",
```

Comment:
This needs to be "Bucket.$" otherwise it's trying to read from a bucket called "$.assets.bucket"